src/entity/Alerts.ts

```diff
@@ -44,6 +44,9 @@ export class Alerts {
   @Column({ type: 'bool', default: false })
   showStreakMilestone: boolean;
 
+  @Column({ type: 'bool', default: false })
+  showResetStreak: boolean;
```

Comment:
If we are saving to redis do we need this? Feels like we would need one or the other no point in saving on multiple places?

Reply:
Might be missing some context but just feels wrong to have two sources of truth.

Reply:
Yeah, at first, I thought this wasn't needed too. But the cache is just to hold the value, which can be viewed from the popup many times until it expires.

This one should hold the value of whether the popup should be shown after boot. Then should not be shown again once seen.